### PR TITLE
Add Google News search command

### PR DESCRIPTION
**Pull request by rlrh**
Follow up to #107

Builds upon existing `FeedCommand`, inspired by `BingWebSearchCommand`

Command word: `googlenews`
Command alias: `gnews`
`gnews` returns top stories
`gnews [KEYWORD] [MORE_KEYWORDS]...` returns news search

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/EntryBookParser.java [hunk 1]
```diff
@@ -6,15 +6,7 @@
...
-import seedu.address.logic.commands.ArchivesCommand;
-import seedu.address.logic.commands.BingWebSearchCommand;
-import seedu.address.logic.commands.Command;
-import seedu.address.logic.commands.ExitCommand;
-import seedu.address.logic.commands.FeedCommand;
-import seedu.address.logic.commands.HelpCommand;
-import seedu.address.logic.commands.HistoryCommand;
-import seedu.address.logic.commands.ListCommand;
-import seedu.address.logic.commands.SelectCommand;
+import seedu.address.logic.commands.*;
```
Comment (thomastanck) [inline src/main/java/seedu/address/logic/parser/EntryBookParser.java]: Don't use star imports

**Review by thomastanck — DISMISSED** [review]
Other than minor style issue, LGTM.

File: src/main/java/seedu/address/logic/parser/EntryBookParser.java (hunk 1 above)
Comment (rlrh) [reply src/main/java/seedu/address/logic/parser/EntryBookParser.java]: Oops, must be IntelliJ who collapsed the imports

Comment (rlrh) [reply src/main/java/seedu/address/logic/parser/EntryBookParser.java]: Fixed in 4b49c71

**Review by qjqqyy — APPROVED** [review]
looks gtm

**Pull request merged**